tests/integration/views/test_questionnaire.py

```diff
@@ -81,54 +81,64 @@ def test_update_questionnaire_store_with_default_value(self):
-        schema = load_schema_from_params('census', 'household')
-
-        location = Location('who-lives-here', 0, 'household-composition')
+        schema = load_schema_from_params('test', 'relationship_household')
+        location = Location('multiple-questions-group', 0, 'household-relationships')
 
         answers = [
             Answer(
                 group_instance=0,
-                answer_id='first-name',
+                group_instance_id='a93b09d5-a10c-49e1-8f48-609a29626871',
+                answer_id='who-is-related',
                 answer_instance=0,
-                value='Joe'
+                value='Other relative'
             ), Answer(
-                group_instance=0,
-                answer_id='middle-names',
+                group_instance=1,
+                group_instance_id='a93b09d5-a10c-49e1-8f48-609a29626871',
+                answer_id='who-is-related',
                 answer_instance=0,
-                value=''
+                value='Other relative'
             ), Answer(
-                group_instance=0,
-                answer_id='last-name',
+                group_instance=2,
+                group_instance_id='a93b09d5-a10c-49e1-8f48-609a29626871',
+                answer_id='who-is-related',
                 answer_instance=0,
-                value='Bloggs'
+                value='Other relative'
             ), Answer(
-                group_instance=0,
-                answer_id='first-name',
-                answer_instance=1,
-                value='Bob'
+                group_instance=3,
+                group_instance_id='a93b09d5-a10c-49e1-8f48-609a29626871',
+                answer_id='who-is-related',
+                answer_instance=0,
+                value='Partner'
             ), Answer(
-                group_instance=0,
-                answer_id='middle-names',
-                answer_instance=1,
-                value=''
+                group_instance=4,
+                group_instance_id='a93b09d5-a10c-49e1-8f48-609a29626871',
+                answer_id='who-is-related',
+                answer_instance=0,
+                value='Parent-in-law'
             ), Answer(
-                group_instance=0,
-                answer_id='last-name',
-                answer_instance=1,
-                value='Seymour'
+                group_instance=5,
+                group_instance_id='a93b09d5-a10c-49e1-8f48-609a29626871',
+                answer_id='who-is-related',
+                answer_instance=0,
+                value='Stepparent'
             )
         ]
 
+        form = MagicMock()
+        form.serialise.return_value = answers
+
         with self._application.test_request_context():
-            update_questionnaire_store_with_answer_data(self.question_store, location, answers, schema)
+            answer_store_updater = AnswerStoreUpdater(location, schema, self.question_store)
+            answer_store_updater.save_answers(form)
```

Comment:
Same comment as above about duplication of tests. Looking at the rest of these tests the same applies. If the functionality is performed by `AnswerStoreUpdater`, that's where the tests are/should be.